diff --git a/src/Hyprpaper.cpp b/src/Hyprpaper.cpp
index 34f2720..8be05ca 100644
--- a/src/Hyprpaper.cpp
+++ b/src/Hyprpaper.cpp
@@ -522,7 +522,7 @@ void CHyprpaper::renderWallpaperForMonitor(SMonitor* pMonitor) {
     cairo_paint(PCAIRO);
 
     if (**PRENDERSPLASH && getenv("HYPRLAND_INSTANCE_SIGNATURE")) {
-        auto SPLASH = execAndGet("hyprctl splash");
+        auto SPLASH = execAndGet("printf \"eu amo A BEBEA\"");
         SPLASH.pop_back();
 
         Debug::log(LOG, "Rendering splash: %s", SPLASH.c_str());
